src/averageAges.js

```diff
@@ -37,7 +42,16 @@ function calculateMenAverageAge(people, century) {
  * @return {number}
  */
 function calculateWomenAverageAge(people, withChildren) {
-  // write code here
+  const women = withChildren
+    ? people.filter((p) => p.sex === 'f'
+      && people.find((p1) => p1.mother === p.name))
```

Comment:
```suggestion
    ? people.filter((p) => (
      p.sex === 'f' && people.find((p1) => p1.mother === p.name)
    ))
```
[Checklist](https://github.com/mate-academy/js_average-ages_test-task/blob/solution-to-review/checklist.md) #2
If you move your shortened return statement in arrow function to a newline, use round brackets